### DISCUSSION TITLE
Espressif SoCs and LCD screens: trends and solutions · Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#130] [General]
# Espressif SoCs and LCD screens: trends and solutions · Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/espressif-socs-lcd-screens/

**Comment by Kevin109** [comment]
Can this RK040BF252（https://www.rocktech.com.hk/lcd-product/rk040bf252/ ） LCD module be used directly with this motherboard?

**Reply by zztiger123** [reply]
Hi Kevin,

This display uses the ST7703 driver IC, and we already have a ready-to-use driver component available here:

https://components.espressif.com/components/espressif/esp_lcd_st7703/versions/2.0.2/readme

It can be used directly once configured with the specific initialization parameters for your screen module.

However, when connecting it to the development board, please check if the pinout matches. Based on our current check, the pin sequences are different, so an additional adapter board may be required.
